lib/axon_onnx/deserialize.ex

```diff
@@ -1908,7 +1910,10 @@ defmodule AxonOnnx.Deserialize do
     shape = random_options["shape"]
     nx_type = onnx_type_to_nx_type(dtype)
 
-    tensor = Nx.random_uniform(List.to_tuple(shape), low, high, type: nx_type)
+    {tensor, _key} =
+      Nx.Random.key(@random_key)
```

Comment:
Instead of this random key, you can use the "seed" option a couple lines above! There is even a TODO. :)

```suggestion
      Nx.Random.key(seed)
```